content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -77,8 +77,8 @@ The **flowcontrol.apiserver.k8s.io/v1beta1** API version of FlowSchema and Prior
 The **autoscaling/v2beta2** API version of HorizontalPodAutoscaler is no longer served as of v1.26.
 
 * Migrate manifests and API clients to use the **autoscaling/v2** API version, available since v1.23.
-* All existing persisted objects are accessible via the new API
```

Comment:
Why remove this line?

Reply:
Initially, I thought the phase was incorrect because the `targetAverageUtilization` field could not be used in the new version. However, upon checking the document again, I found no mention of this field being persisted. Therefore, my previous assumption was still correct, and I had simply forgotten to revert the removal of that field.

Reply:
Updated.